clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -146,6 +146,14 @@ static bool startsNextParameter(const FormatToken &Current,
            Style.BreakInheritanceList != FormatStyle::BILS_BeforeComma));
 }
 
+// Returns \c true if \c Current starts a new operand in a binary operation.
+static bool startsNextOperand(const FormatToken &Current) {
+  const FormatToken &Previous = *Current.Previous;
```

Comment:
Assert `Current.Previous` before dereferencing it.
```suggestion
  const auto &Previous = *Current.Previous;
```